src/briefcase/commands/base.py

```diff
@@ -27,7 +27,7 @@
 if sys.version_info >= (3, 11):  # pragma: no-cover-if-lt-py311
     import tomllib
 else:  # pragma: no-cover-if-gte-py311
-    import tomli as tomllib
+    import tomli as tomllib  # pragma: no cover:
```

Comment:
Why is this needed? The `no-cover-if-gte-py311` should be sufficient here
```suggestion
    import tomli as tomllib
```

Reply:
Yeah my bad, I saw a coverage failure in my branch and initially added the pragma out of caution, but the existing no-cover-if-gte-py311 logic already handles this case. I am definitely learning a lot about testing and coverage with these PRs.